index.js

```diff
@@ -8,10 +9,24 @@ const parser = require("./src/parser");
 const printDocToDebug = require("./src/doc-debug").printDocToDebug;
 
 function parse(text, opts) {
-  if (opts.parser === "flow") {
-    return parser.parseWithFlow(text, opts.filename);
+  const parseFunction = opts.parser === "flow"
+    ? parser.parseWithFlow
+    : parser.parseWithBabylon;
+
+  try {
+    return parseFunction(text);
+  } catch (error) {
+    const loc = error.loc;
+
+    if (loc) {
+      error.codeFrame = codeFrame(text, loc.line, loc.column + 1, {
+        highlightCode: true
```

Comment:
Re: browser version, think we need to add `rollup-plugin-node-builtins` to shim `process` for all the checks in chalk/etc, otherwise, works great:

![image](https://cloud.githubusercontent.com/assets/56288/22179378/3e54e502-e017-11e6-8e96-b2e3c4737520.png)